dll/win32/kernel32/client/thread.c

```diff
@@ -1092,240 +1092,136 @@ QueueUserWorkItem(IN LPTHREAD_START_ROUTINE Function,
     return TRUE;
 }
 
-/*
- * @implemented
- */
-DWORD
-WINAPI
-TlsAlloc(VOID)
```

Comment:
What about keeping the old implementation within some
`#if (_WIN32_WINNT < _WIN32_WINNT_VISTA) && (DLL_EXPORT_VERSION < _WIN32_WINNT_VISTA)`
block?

Reply:
I will do it then , better if u can help me with that, excellent idea , both NT5 & NT6 version remains

Reply:
For that, look at the original code and yours, and see where there are actual functional differences.